stackit/internal/services/iaas/iaas_acc_test.go

```diff
@@ -87,40 +144,157 @@ func networkAreaRouteResourceConfig() string {
 	)
 }
 
-func resourceConfig(name, nameservers, areaname, networkranges string) string {
-	return fmt.Sprintf("%s\n\n%s\n\n%s\n\n%s",
+func networkInterfaceResourceConfig(name string) string {
+	return fmt.Sprintf(`
+				resource "stackit_network_interface" "network_interface" {
+					project_id = stackit_network.network.project_id
+					network_id = stackit_network.network.network_id
+					name       = "%s"
+				}
+				`,
+		name,
+	)
+}
+
+func volumeResourceConfig(name, size string) string {
+	return fmt.Sprintf(`
+				resource "stackit_volume" "volume" {
+					project_id = "%s"
+					availability_zone = "%s"
+					name = "%s"
+					description = "%s"
+					size = %s
+					labels = {
+						"label1" = "%s"
+					}
+					performance_class = "%s"
+				}
+				`,
+		volumeResource["project_id"],
+		volumeResource["availability_zone"],
+		name,
+		volumeResource["description"],
+		size,
+		volumeResource["label1"],
+		volumeResource["performance_class"],
+	)
+}
+
+func serverResourceConfig(name, machineType string) string {
+	return fmt.Sprintf(`
+				resource "stackit_server" "server" {
+					project_id = "%s"
+					availability_zone = "%s"
+					name = "%s"
+					machine_type = "%s"
+					boot_volume = {
+						size = %s
+						source_type = "%s"
+						source_id = "%s"
+					}
+					labels = {
+						"label1" = "%s"
+					}
+					user_data = "%s"
+				}
+				`,
+		serverResource["project_id"],
+		serverResource["availability_zone"],
+		name,
+		machineType,
+		serverResource["size"],
+		serverResource["source_type"],
+		serverResource["source_id"],
+		serverResource["label1"],
+		serverResource["user_data"],
+	)
+}
+
+func securityGroupResourceConfig(name string) string {
+	return fmt.Sprintf(`
+				resource "stackit_security_group" "security_group" {
+					project_id = "%s"
+					name = "%s"
+					description = "%s"
+					labels = {
+						"label1" = "%s"
+					}
+				}
+				`,
+		volumeResource["project_id"],
```

Comment:
The acceptance test works like this. Just for the clarity, I think these need to be changed to `securityGroupResource`. 

This can be done when we are editing acceptance test next week.